### PR TITLE
feat: show user role details on the users list table

### LINKED ISSUE #682: Add a role onto users
- Admin
- Operator
- Vendor
- Viewer

### PR DESCRIPTION
**Pull request by Prashanth-Nelli**
Closes #682 & https://github.com/flanksource/flanksource-ui/issues/1135

### CODE REVIEW

**Comment by moshloop** [comment]
@Prashanth-Nelli please resolve conflicts

**Comment by moshloop** [comment]
@Prashanth-Nelli - I have an empty screen now ?

**Comment by Prashanth-Nelli** [comment]
> @Prashanth-Nelli - I have an empty screen now ?

@moshloop the issue is due to the flag NEXT_PUBLIC_WITHOUT_SESSION being set to true on netlify deployment.

created an issue for this https://github.com/flanksource/flanksource-ui/issues/1135

**Comment by Prashanth-Nelli** [comment]
@moshloop this can be merged now.

File: src/context/UserAccessContext/permissions.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  kratos: "kratos"
+};
+
+export const permDefs = {
+  admin: {
+    read: [...Object.values(tables)],
+    write: [...Object.values(tables)]
+  },
+  editor: {
+    read: [tables.canaries, tables.topologies, tables.config_scrapers],
```
Comment (moshloop) [inline src/context/UserAccessContext/permissions.ts:23]: editor can read everything but connections
and write to everything except connections and users

commander and responder both extend the viewer role
responders have the same table permissions as commanders, but responders cannot update the hypothesis status or change the incident status or commander - these lower level permissions we can however defer

Please also test all the roles - There shouldn't. be any errors shown when navigating

**Comment by Prashanth-Nelli** [comment]
these changes are already merged to main branch through the PR https://github.com/flanksource/flanksource-ui/pull/623

**Pull request closed without merging**